### ISSUE TITLE
No default font found on ArchLinux and no error message (includes patch)

### ISSUE THREAD

**Issue by feinstaub** [pvcraven/arcade#402]
## Bug Report
Starting to work with arcade 2.0.4 via pip3 on ArchLinux but no text is displayed on draw_text.

### What should be added/changed?
This makes it work on my default Manjaro installation:
```
diff --git a/arcade/text.py b/arcade/text.py
index ac16ea6..105d9d4 100644
--- a/arcade/text.py
+++ b/arcade/text.py
@@ -198,6 +198,7 @@ def draw_text(text: str,
         # Default font if no font
         if font is None:
             font_names = ("arial.ttf",
+                          'NotoSans-Regular.ttf',
                           "/usr/share/fonts/truetype/freefont/FreeMono.ttf",
                           '/System/Library/Fonts/SFNSDisplay.ttf')
             for font_string_name in font_names:

```

It would be nice if an error message is printed to the console if no font was found. This is better than just displaying no text.

**Comment by pvcraven** [comment]
Hey thanks for the patch!

**Comment by pvcraven** [comment]
Fixed with 2.0.9 release.
